src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -54,7 +54,14 @@ public void init() throws ServletException {
     setMessageStore(MessageStore.getInstance());
     setUserStore(UserStore.getInstance());
     //this sets the HTML tags that the user will be allowed to use
-    chatOk.addTags("strong","em");
+    chatOk.addTags("div","span","strong","em","del","style","mark","sub","sup","h1","h2","h3","h4","h5","h6","blockquote","cite","dfn","a","img","iframe","code","samp");
```

Comment:
This line seems very long, Google style is 100 characters for Java. It seems pedantic but it turns out to be really nice to set one's IDE to support 2 (or 3 if you have a fantastic monitor) 100-column code panes side by side and be assured of being able to comfortably read all the code.